starkjs/src/pil_verifier.js

```diff
@@ -47,44 +50,7 @@ module.exports = {
     assert(res.length == 0);
 
     elapse("arithmetization", timer);
-    // prove and verify the stark proof
-/*
```

Comment:
We don't have to delete those codes, casue it's still useful to be as a baseline for Rust stark verifier.